web/modules/custom/quiltme_picsart/quiltme_picsart.module

```diff
@@ -154,8 +155,64 @@ function quiltme_picsart_node_presave(NodeInterface $node) {
           $image_file = File::load($image_media->field_media_image->target_id);
           $image_file_url = \Drupal::request()->getSchemeAndHttpHost() . $image_file->createFileUrl();
           $node->set('field_chosen_image_url', $image_file_url);
+          if ($node->get('field_email_sent')->value != 1) {
+            $email_opts = [
+              'to' => $node->get('field_user_email')->value,
+              'image_url' => $image_file_url,
+              'name' => $node->getTitle(),
+            ];
+            quiltme_picsart_send_user_email($email_opts);
+            $node->set('field_email_sent', 1);
+          }
         }
       }
       break;
   }
 }
+
+/**
+ * Custom send email function.
+ */
+function quiltme_picsart_send_user_email(array $options) {
+  $mailManager = \Drupal::service('plugin.manager.mail');
```

Comment:
@laura-johnson with the SMTP addition, will we need to make any adjustments to this section of code?

Reply:
Not sure, I'll give it a test on the multidev.